va/va.go

```diff
@@ -171,6 +170,33 @@ func initMetrics(stats prometheus.Registerer) *vaMetrics {
 	}
 }
 
+// PortConfig specifies what ports the VA should call to on the remote
+// host when performing its checks.
+type PortConfig struct {
```

Comment:
This type no longer needs to be exported
```suggestion
type portConfig struct {
```